### PR TITLE
Added .editorconfig to help standardise code contributions

### PR DESCRIPTION
**Pull request by robinnorth**
In response to #9, and seeing as you're asking for Hacktoberfest help, I've added an `.editorconfig` for you to help contributors standardise their code contributions. Support for the standard is built into many IDEs already, and others are supported with the use of plugins. For example, I use WebStorm which supports it out of the box and Sublime Text which requires a plugin: http://editorconfig.org/#download

If, like me, you're really keen to enforce a consistent programming style as well as low-level text formatting, you should consider adding [ESLint](http://eslint.org/) to the project (or opening an issue for someone else to do it 😉 ), which will do this for you, as well as providing useful syntax error-checking.

### CODE REVIEW

**Comment by robinnorth** [comment]
No problem, @itaisteinherz, you're very welcome 😃

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# Change these settings to your own preference
+indent_style = space
+indent_size = 4
+
+# We recommend you to keep these unchanged
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```
Comment (itaisteinherz) [inline .editorconfig]: I prefer my code without a new line in the end, so it would be great if you could change this to false.

**Comment by itaisteinherz** [comment]
Thanks for the changes! if you could just resolve the conflict with .gitignore, that would be awesome.

**Comment by robinnorth** [comment]
@itaisteinherz, rebased your changes to the `.gitignore` and pushed. Being able to specify your personal preferences is exactly what makes the `.editorconfig` standard so neat!

**Comment by itaisteinherz** [comment]
Thank you very much for your contribution so far, @robinnorth ! If you have any other ideas or suggestions on how to make this project better, you are more than welcome to create another pull request.

**Comment by robinnorth** [comment]
Happy to help, @itaisteinherz !

**Pull request merged**